src/cli.js

```diff
@@ -93,14 +93,11 @@ process.on('uncaughtException', (err) => {
 /**
  * Validates and sets the given flag within runner
  */
-function processFlag(name, value) {
-    try {
-        if(typeof name != 'string') {
-            name = name.toString();
-        }
+function processFlag(name = name.toString(), value) {
+    try { 
```

Comment:
I'm kind of opposed to any default params that aren't a simple value like `0`. Especially here, where I'd like `name` and `value` to be initialized similarly

Reply:
Ok I reverted this change. Everything should be good now, let me know what you think.